config/test.json

```diff
@@ -166,6 +172,18 @@
       "<p>#LINK#</p>",
       "<hr>"
     ],
+    "expired": [
+        "User #UID# has expired, updating account"
+    ],
+    "expired_html": [
+        "<p>User #UID# has expired, updating account</p>"
+    ],
+    "project_expiration": [
+        "Project quota for #NAME# will expire at #DATE#"
```

Comment:
should just talk about project expiration, not quota, project may be more than a disk extension

Reply:
https://github.com/genouest/genouestaccountmanager/pull/166/commits/aebfa84ebba26c26dc21888b0836aba0b3cd352a